src/libs/elektra/mount.c

```diff
@@ -362,11 +363,6 @@ int elektraMountModules (KDB * kdb, KeySet * modules, Key * errorKey)
 
 	while ((cur = ksNext (modules)) != 0)
 	{
-		// exclude "virtual" plugins where actual plugins could not deliever
-		// their contract properly
-		if (!strcmp (keyName (cur), "system/elektra/modules/resolver")) continue;
```

Comment:
sorry, i really need to improve my flow of work with multiple buffers and branches ...